src/index.ts

```diff
@@ -183,9 +183,7 @@ if (argv) {
         app = new Application(false, silence, artie);
       }
       app.startApplication(argv);
-    } catch (error) {
-      console.error(error.message);
-    }
+    } catch (error) {}
```

Comment:
empty catch?

Reply:
🤷‍♂

Reply:
Yeah, I dunno why I took out the console.error, etc..., added that back in, great catch!